bin/check-jenkins-job-status.rb

```diff
@@ -110,9 +110,10 @@ def job_status(job_name)
     status = jenkins_api_client.job.get_current_build_status(job_name)
     # If the job is currently running, get the status of the last build instead
     if status == 'running'
-      last_build = jenkins_api_client.job.get_current_build_number(job_name) - 1
-      build = jenkins_api_client.job.get_build_details(job_name, last_build)
-      status = build['result'].downcase
+      build = jenkins_api_client.job.get_build_details(job_name, 'lastCompletedBuild')
+      if build and build['result']
```

Comment:
please use `&&` over `and` it has to do with the precedence of operators

see: https://www.tinfoilsecurity.com/blog/ruby-demystified-and-vs

Reply:
I am also thinking we should have an else so that we can return `false` over `nil` which makes it easier to figure out whats going on if we do ever have a similar issue again.

Reply:
It would not return nil but 'running', which seems appropriate since there would be no 'lastCompletedBuild' and the last build is running.
So we can assume that it's a new job (or existing job with an log rotator that removed the last completed build for instance), and so the last build that the script has access too is currently being 'build', so the 'running' status seems appropriate to me.